### PR TITLE
Add initial 1.5.0 benchmark

### PR DESCRIPTION
**Pull request by jiaodong**
Some files are simply missing results or not reproducible locally. Looking for more input as I didn't see docs about generating them yet ..

stream shuffle for example:

```
❯ python -m ray.experimental.shuffle --num-cpus=32 --num-partitions=200 --partition-size=500e6 --object-store-memory=20e9
Start a new cluster...
2021-08-02 11:09:42,860	INFO services.py:1245 -- View the Ray dashboard at http://127.0.0.1:8265
Traceback (most recent call last):
  File "/Users/jiaodong/anaconda3/envs/ray/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/Users/jiaodong/anaconda3/envs/ray/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/jiaodong/Workspace/ray/python/ray/experimental/shuffle.py", line 318, in <module>
    main()
  File "/Users/jiaodong/Workspace/ray/python/ray/experimental/shuffle.py", line 258, in main
    tracker = _StatusTracker.remote()
  File "/Users/jiaodong/Workspace/ray/python/ray/actor.py", line 423, in remote
    return self._remote(args=args, kwargs=kwargs)
  File "/Users/jiaodong/Workspace/ray/python/ray/util/tracing/tracing_helper.py", line 367, in _invocation_actor_class_remote_span
    return method(self, args, kwargs, *_args, **_kwargs)
  File "/Users/jiaodong/Workspace/ray/python/ray/actor.py", line 726, in _remote
    actor_id = worker.core_worker.create_actor(
  File "python/ray/_raylet.pyx", line 1320, in ray._raylet.CoreWorker.create_actor
TypeError: create_actor() takes exactly 18 positional arguments (14 given)
```

@wuisawesome and @rkooo567 helped to confirm distributed test and shuffle logs can be skipped.

@richardliaw @sven1977  same for rllib 

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: release/release_logs/1.5.0/microbenchmark.txt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+single client tasks sync per second 1306.99 +- 5.19
+single client tasks async per second 10444.40 +- 322.83
+multi client tasks async per second 34150.36 +- 618.02
+1:1 actor calls sync per second 2148.63 +- 16.30
+1:1 actor calls async per second 5778.55 +- 211.10
+1:1 actor calls concurrent per second 5304.30 +- 54.80
+1:n actor calls async per second 14886.74 +- 780.54
+n:n actor calls async per second 42208.40 +- 1476.76
+n:n actor calls with arg async per second 6563.52 +- 401.34
+1:1 async-actor calls sync per second 1351.72 +- 21.58
```
Comment (rkooo567) [inline release/release_logs/1.5.0/microbenchmark.txt:18]: Can you post the percentage change to the release channel? There must be a script to do this. `microbenchmark_analysis.py`

**Review by rkooo567 — APPROVED** [review]
LGTM! Can you compare to the previous versions and let us know if there were regressions? It'll be also great if you can update this to the release doc update (e.g., this needs to be reported before release goes out).

**Comment by rkooo567** [comment]
Streaming shuffle is not needed. Test failures should be unrelated

File: release/release_logs/1.5.0/microbenchmark.txt (hunk 1 above)
Comment (jiaodong) [reply release/release_logs/1.5.0/microbenchmark.txt:18]: sure, i noticed a bit regression on this while writing this file, i can run the script tomorrow sometime and flag that to ray core.

**Pull request merged**